scripts/dqlite/scripts/dqlite/dqlite-build.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+set -e
+
+build() {
```

Comment:
If my changes here:
https://github.com/juju/juju/pull/15147

could be integrated that would be great